build.gradle

```diff
@@ -13,7 +13,7 @@
 
 buildscript {
     ext.ionVersion = "[1.2,)"
-    ext.kotlinVersion = "[1.2,)"
+    ext.kotlinVersion = "[1.2,1.6["
```

Comment:
Added this because the CI build fails for kotlin version 1.6.0-M1 to only allow stable kotlin versions.